frontend/src/components/Account/MemberAccount/index.js

```diff
@@ -0,0 +1,78 @@
+import React, { useState, useEffect } from "react";
+import PropTypes from "prop-types";
+import { useLocation } from "react-router-dom";
+import { Select } from "@code4ro/taskforce-fe-components";
+import "./MemberAccount.scss";
+
+import mockData from "../mockData/mockData";
+import ProfileHistory from "../common/ProfileHistory/ProfileHistory.js";
+
+export const MemberAccount = ({ data }) => {
+  const location = useLocation();
+  const familyMembers = data;
+  const defaultMember = familyMembers.length > 0 ? familyMembers[0] : undefined;
+  const [selectedMember, setSelectedMember] = useState(defaultMember);
+  const [options, setOptions] = useState([]);
+
+  useEffect(() => {
+    const queryParams = location.search;
+    // todo: use a library to read the query parameter in a more robust way
+    const personId =
+      !!queryParams && queryParams.indexOf("personId") !== -1
```

Comment:
this will probably break if we have more than one query param ...

Reply:
added querystring library to handle multiple parameters